docs/install/tkgs.rst

```diff
@@ -29,9 +29,26 @@ For the deployment on TKG clusters, Kubeflow on vSphere is installed on a Tanzu
 
 - Install ``kctrl``, a kapp-controller's native CLI on your client host. It is used to install  Kubeflow on vSphere Carvel Package. See `Installing kapp-controller CLI: kctrl <https://carvel.dev/kapp-controller/docs/v0.40.0/install/#installing-kapp-controller-cli-kctrl>`__.
 
+Minimus resources for TKG cluster to Install Kubeflow
+======================================================
+
+The TKG cluster must meet the following minimu reqyurments:
+
+- Kubernetes version 1.21 or later
+- At least one worker node with a minimum of:
+    - 4 CPU
+    - 16 GB memory
+    - 50 GB storage
+
+Also note: the minimus resources of TKG cluster can install Kubeflow successfully, but it's more like a toy. For workloads with large resources, Kubeflow may not deploy your workloads due to the limited resources.
+
+Thus, it depends on the workloads to create suitable resources TKG cluster to deploy Kubeflow.
+
 Deploy Kubeflow on vSphere package on TKG clusters
 ===========================================================
 
+The below deployment procedure is based on Linux. For the Windows user, the deployment procedure is the same as below, just need to pay attention to install Windows version of the `kubectl`, `kctrl` software.
```

Comment:
Note that the below deployment procedure is for Linux and Windows users, but Windows users would need to first install the Windows version of ``kubectl`` and ``kctrl`` command(?).